root.go

```diff
@@ -36,7 +36,8 @@ including building WebAssembly Runnables and Atmo projects.`,
 
 	create.AddCommand(command.CreateProjectCmd())
 	create.AddCommand(command.CreateRunnableCmd())
-	create.AddCommand(command.CreateHandlerCmd())
+	// TODO: turn into create workflow command
```

Comment:
Do we have an issue for this we can reference here?

Reply:
Added an issue reference